libobs/media-io/video-matrices.c

```diff
@@ -247,6 +247,8 @@ bool video_format_get_parameters(enum video_colorspace color_space,
 	case VIDEO_CS_2100_PQ:
 	case VIDEO_CS_2100_HLG:
 		bpc = 10;
+	default:
+		break;
```

Comment:
- If adding some code in future, we should avoid fall-through.
- If adding new enum elements for `video_colorspace`, we should list all cases so that the warning can catch newly added elements.
```suggestion
		break;
	case VIDEO_CS_DEFAULT:
	case VIDEO_CS_601:
	case VIDEO_CS_709:
	case VIDEO_CS_SRGB:
		bpc = 8;
```

Reply:
The warning was literally removed in libobs. So it means that we don't care about managing every member of the enumeration.